plat/socionext/synquacer/drivers/scmi/sq_scmi_private.h

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2019, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef SQ_SCMI_PRIVATE_H
+#define SQ_SCMI_PRIVATE_H
+
+#include <lib/mmio.h>
+
+/* SCMI vendor specific protocol */
+#define SCMI_SYS_VENDOR_EXT_PROTO_ID		0x80
```

Comment:
This is the only difference from scmi_private.h. Would suggest to add this to the scmi_private.h and remove this file.